spec/fixtures/katello-certs-check/passphrase-onkey.txt

```diff
@@ -0,0 +1 @@
+The /home/travis/build/theforeman/foreman-installer/spec/fixtures/katello-certs-check/key_pass.key contains a passphrase, please recreate the key without a passphrase by doing: cp -a /home/travis/build/theforeman/foreman-installer/spec/fixtures/katello-certs-check/key_pass.key /home/travis/build/theforeman/foreman-installer/spec/fixtures/katello-certs-check/key_pass.key.old openssl rsa -in /home/travis/build/theforeman/foreman-installer/spec/fixtures/katello-certs-check/key_pass.key.old -out /home/travis/build/theforeman/foreman-installer/spec/fixtures/katello-certs-check/key_pass.key
```

Comment:
These absolute paths will mean that a developer can never have passing builds. Probably best to copy the pattern of replacing `|COMMAND|` but then for the `directory`.